### ISSUE TITLE
Cannot list releases

### ISSUE THREAD

**Issue by ludwikbukowski** [yrashk/kerl#112]
I started using kerl some time ago and it was fine but today I needed to build Erlang 18.2.
I tried to update my releases list but had some troubles. Logs below:

```
Administrators-MacBook-Pro:~ ludwikbukowski$ kerl list releases

Run "/Users/ludwikbukowski/Installed/kerl/kerl/kerl update releases" to update this list from erlang.org
Administrators-MacBook-Pro:~ ludwikbukowski$ kerl update releases
Getting the available releases from erlang.org...
The available releases are:

Administrators-MacBook-Pro:~ ludwikbukowski$ kerl list releases

Run "/Users/ludwikbukowski/Installed/kerl/kerl/kerl update releases" to update this list from erlang.org
Administrators-MacBook-Pro:~ ludwikbukowski$
```

**Comment by darrencauthon** [comment]
For whatever it's worth, I just started getting this as well.

**Comment by darrencauthon** [comment]
Also, for whatever it's worth... I blasted away my ```.kerl``` directory and started fresh.  It all seems to work.

**Comment by fpauser** [comment]
Same problem here, using korora(fedora) - "blasting away .kerl" did not help.

**Comment by fooflare** [comment]
Hi,
I checked in Debian with the master branch and is working.

**Comment by darrencauthon** [comment]
I should have been more clear... 

"blasted away my ```.kerl" means:  

```
cd ~
rm -rf .kerl
```

Then I reinstalled kerl using the instructions in the readme, and I had a working kerl again.

**Comment by fpauser** [comment]
Ah, yes, that solved it for me too. This issue can be closed from my side.

**Comment by brucify** [comment]
Did https://github.com/yrashk/kerl/pull/107 solve this problem?

**Comment by fpauser** [comment]
I would say yes.

**Comment by mrallen1** [comment]
It looks like this issue has been resolved by #107. If not, please open a new ticket
